### PR TITLE
Add typing for qsConfig method to url-assembler

### PR DESCRIPTION
**Pull request by alicewriteswrongs**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/Floby/node-url-assembler/blob/master/lib/url-assembler-factory.js#L65>>

### CODE REVIEW

**Comment by wolfgang42** [comment]
[Looks like](https://github.com/Floby/node-url-assembler/commit/3728dbc0adca0d6c6dd9eeb01b132e8e58cf8ca5) this method was added in `url-assembler@2.1.0`, so the version in the header should be updated. You might also consider adding yourself to the authors list as well.

Otherwise LGTM, but I'll hold off on hitting "Approve" since I'm not sure what the automation will do with that and it'd be silly to do another PR just to bump the version header.

**Comment by alicewriteswrongs** [comment]
@wolfgang42 thanks for taking a look! I just made the changes you suggested. Managed to miss the version in the header initially!

File: types/url-assembler/index.d.ts [hunk 1]
```diff
@@ -1,19 +1,22 @@
-// Type definitions for url-assembler 1.2
+// Type definitions for url-assembler 2.1.0
```
Comment (wolfgang42) [inline types/url-assembler/index.d.ts:1]:
```suggestion
// Type definitions for url-assembler 2.1
```

To fix the CI failure; took me a bit to work out where to find the actual log but it looks like the error is:

> ERROR: 1:25  dt-header  Error parsing header. Expected: foo MAJOR.MINOR (patch version not allowed). See: https://github.com/Microsoft/dtslint/blob/master/docs/dt-header.md

Comment (alicewriteswrongs) [reply types/url-assembler/index.d.ts:1]: ahh thanks for finding the issue! just pushed the change

**Review by wolfgang42 — APPROVED** [review]

**Comment by alicewriteswrongs** [comment]
Ready to merge

**Comment by alicewriteswrongs** [comment]
@wolfgang42 thanks for your help!

**Pull request merged**